packages/cli/src/commands/server/runServer.js

```diff
@@ -19,6 +19,7 @@ import webSocketProxy from './webSocketProxy';
 import MiddlewareManager from './middleware/MiddlewareManager';
 
 import loadMetroConfig from '../../tools/loadMetroConfig';
+import logger from '../../../../tools/src/logger';
```

Comment:
```suggestion
import {logger} from '@react-native-community/cli-tools';
```

Reply:
@thymikee did not get a chance fixing this yesterday. thanks for helping